docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -49,7 +49,7 @@ x-airflow-common:
   environment:
     &airflow-common-env
     AIRFLOW__CORE__EXECUTOR: CeleryExecutor
-    AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql+psycopg2://airflow:airflow@postgres/airflow
```

Comment:
This file will only be compatible with Airflow 2.3.0 now. Can you update the check on line 175 or add backward compatibility(preferred)?

Reply:
Thanks a lot @mik-laj . I believe the deprecation support we added will handle this. See changes in [configuration.py](https://github.com/apache/airflow/blob/45a352e41692e22b7d161dc9b0c85bbf228d8f56/airflow/configuration.py#L190) file.
If these won''t be good can you help suggest what we need to do to get this fully backward compatible ?

Reply:
In other words, I would like the following command to work and not display any warnings on any version.
```
AIRFLOW_IMAGE_NAME=apache/airflow:2.2.3 docker-compose up
```
If this is not possible, we should display a message that this version is not supported.